registration-processor/stage-groups/pom.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+<project
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+	xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<modelVersion>4.0.0</modelVersion>
+	<packaging>pom</packaging>
+	<parent>
+		<groupId>io.mosip.registrationprocessor</groupId>
+		<artifactId>registration-processor</artifactId>
+		<version>1.2.0-SNAPSHOT</version>
+	</parent>
+	<version>1.2.0-SNAPSHOT</version>
+	<artifactId>stage-groups</artifactId>
+	<name>stage-groups</name>
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+	</properties>
+	<modules>
+		<module>group-1</module>
+		<module>group-2</module>
+		<module>group-3</module>
+		<module>group-4</module>
+		<module>group-5</module>
+		<module>group-6</module>
+		<module>group-7</module>
+    </modules>
+    <dependencies>
+		<dependency>
+			<groupId>io.micrometer</groupId>
+			<artifactId>micrometer-core</artifactId>
+			<version>${io.micrometer.prometheus.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>io.micrometer</groupId>
+			<artifactId>micrometer-registry-prometheus</artifactId>
+			<version>${io.micrometer.prometheus.version}</version>
+		</dependency>
+    </dependencies>
+</project>
```

Comment:
Few extra queries:
1. How does a developer run the a stage in local ?
2. When the develop runs the stage group in local, can he use a property file rather than configuring the config server?
3. @ComponentScan(basePackages - overlap is taken care by the spring ?

Reply:
1. To run one ore more stage (stage group), 

- Have the local config server running.
- Add the stage project to the build path of mosip-stage-executor (in eclipse) and launch the MosipStageExecutorApplication pointing to the local config server, 
- Or, run the stage group as in below example command:

`java -Dapplication.base.url=http://localhost:8090 -Dspring.profiles.active=mz -Dspring.cloud.config.uri=http://localhost:51000/config -Dspring.cloud.config.label=master  -Dstage-group-name=stage-group-1 -jar registration-processor-stage-group-1-1.2.0-SNAPSHOT.jar`

2. Running without config server and only the local applicatoin.properties does not work since mosip stage executor is fetching the configuration only from config server and it is not loading the local application properties.